code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -284,7 +284,7 @@
 	return 1
 
 /datum/species/proc/handle_temperature(datum/gas_mixture/breath, var/mob/living/carbon/human/H) // called by human/life, handles temperatures
-	if( (abs(310.15 - breath.temperature) > 50) && !(RESIST_HEAT in H.mutations)) // Hot air hurts :(
+	if(abs(310.15 - breath.temperature) > 50) // Hot air hurts :(
```

Comment:
only reason i'd disagree is because it means resist heat won't actually help you run through fire